nftables.go

```diff
@@ -463,6 +471,11 @@ func parseElementValue(json interface{}) ([]string, error) {
 				}
 			}
 			return vals, nil
+		} else if prefix, _ := jsonVal[map[string]interface{}](val, "prefix"); prefix != nil {
+			// For prefix-type elements, return the element in CIDR representation.
+			addr, _ := jsonVal[string](prefix, "addr")
+			length, _ := jsonVal[float64](prefix, "len")
+			return []string{fmt.Sprintf("%s/%d", addr, int(length))}, nil
```

Comment:
@danwinship is on purpose we  swallow all the parsing errors in a library?

Reply:
the general philosophy here is:
1. don't crash
2. if we get bad data for a field that we are parsing, return an error
3. but assume that `nft` might return additional fields in some objects in the future

so yeah, if you don't actually get a string and a float here, return an error

Reply:
Thanks, I've updated this to check and return errors now :+1: